Clojure/Clojure.sublime-settings

```diff
@@ -0,0 +1,7 @@
+{
+  "word_separators": "/\\()[]{}\"'`,:;@#^~.",
+  "smart_indent": false,
+  "detect_indentation": false,
+  "tab_size": 1,
+  "draw_indent_guides": false
```

Comment:
Is this related to the tab size of 1?

Reply:
Yeah. There's a reason for these settings.

* `smart_indent` doesn't play well with Lisp code
* `detect_indentation` can get confused by traditional Clojure indentation, and there's very rarely any reason for indent other than 1
* `draw_indent_guides` creates needless noise with indent 1